mobilizon_bots/models/event.py

```diff
@@ -3,16 +3,27 @@
 
 
 class Event(Model):
-    id = fields.UUID(pk=True)
+    id = fields.UUIDField(pk=True)
     name = fields.TextField()
+    description = fields.TextField()
 
-    # tournament = fields.ForeignKeyField('models.Tournament', related_name='events')
-    # participants = fields.ManyToManyField('models.Team', related_name='events', through='event_team')
-    # modified = fields.DatetimeField(auto_now=True)
-    # prize = fields.DecimalField(max_digits=10, decimal_places=2, null=True)
+    mobilizon_id = fields.TextField()
+    mobilizon_link = fields.TextField()
+    thumbnail_link = fields.TextField(null=True)
+
+    location = fields.TextField(null=True)
+
+    begin_datetime = fields.DatetimeField()
+    end_datetime = fields.DatetimeField()
+    # UTC offset in seconds
+    utcoffset = fields.IntField()
```

Comment:
why is this necessary? Doesn't "DateTimeField" already store the timezone? Also what does this refer to?

Reply:
I think this is best discussed synchronously. Tortoise does support timezones but you can either set a global timezone that'll be used when selecting datetimes or just store everything as UTC. 

https://tortoise-orm.readthedocs.io/en/latest/timezone.html

I believe the second is the most flexible solution as it'll allow us to store different events with different timezones (but I'm not sure about actual use cases for different timezones)